mythtv/libs/libmythupnp/eventing.cpp

```diff
@@ -75,7 +77,7 @@ Eventing::Eventing(const QString &sExtensionName,
     HttpServerExtension(sExtensionName, sSharePath),
     m_sEventMethodName(std::move(sEventMethodName)),
     m_nSubscriptionDuration(
-        UPnp::GetConfiguration()->GetDuration<std::chrono::seconds>("UPnP/SubscriptionDuration", 30min))
+        GetMythDB()->GetDuration<std::chrono::seconds>("UPnP/SubscriptionDuration", 30min))
```

Comment:
This isn't correct. The database does not contain any setting for "UPnP/SubscriptionDuration". This is *only* (optionally) present in config.xml.

Reply:
The [wiki](https://www.mythtv.org/wiki/Config.xml) doesn't mention these settings, so I thought they would be in the database.  Only UPnP/UDN/MediaRenderer was ever saved by the code.

Should these settings be saved to config.xml?

Reply:
Since this is read only, would `XmlConfiguration("config.xml").Get …` or equivalent be acceptable?
```
auto config = XmlConfiguration("config.xml");
foobar = config.Get …
```